cypress/e2e/generateExcel.cy.js

```diff
@@ -0,0 +1,53 @@
+const testURLRoot = "http://127.0.0.1:50722/templates/";
+const fixture = "json/templates_generated.json";
+const uuid = "396fc51e-dd55-451f-a6dd-913177abb667";
+const project = "enanomapper";
+
+describe("Generate Excel File", () => {
+  beforeEach(() => {
+    cy.visit(testURLRoot);
+    cy.intercept(
+      {
+        method: "GET",
+        url: "/template",
+        hostname: "api-test.ramanchada.ideaconsult.net",
+      },
+      {
+        fixture,
```

Comment:
For consistency, please either define the fixture directly here, or define the fixture of the next intercept via a constant as well.